docs/quickstart/aws.md

```diff
@@ -0,0 +1,152 @@
+```
+
+This will return a series of Coder logs, however, embedded in the launch is the URL for accessing Coder.
+
+<img src="../images/quickstart/aws/aws8.png">
+
+In this instance, Coder can be accessed at the url `https://fccad1b6c901511b30cf2cf4fbd0973e.pit-1.try.coder.app`.
+
+Copy the URL and run the following command to create the first user, either on your local machine or in the AWS EC2 instance terminal. 
+
+```sh
+coder login <url***.try.coder.app>
+```
+
+Fill out the prompts. Be sure to save use email and password as these are your admin username and password.
+
+You can now access Coder on your local machine with the relevant  `***.try.coder.app` URL and logging in with the username and password.
+
+## Creating and Uploading Your First Template
+
+Run `coder template init` to create your first template. You’ll be given a list of possible templates. This tutorial will show you how to set up your Coder instance to create Linux based machines on AWS. 
+
+<img src="../images/quickstart/aws/aws9.png">
+
+Press `enter` to select `Develop in Linux` on AWS template. This will return the following: 
+
+<img src="../images/quickstart/aws/aws10.png">
+
+Now, we must install the AWS CLI and authorize the template. Follow [these instructions to install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) and [add your credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html).
+
+Coder runs as a system service under a system user `coder`. The `coder` user will require access to the AWS credentials to initialize the template and provision workspaces.
+
+Run the following command to create a folder for the AWS credentials to live in:
+
+```sh
+sudo mkdir /home/coder/.aws
+```
+
+Run the following commands to copy the AWS credentials and give the `coder` user access to them:
+
+```sh
+sudo cp ~/.aws/credentials /home/coder/.aws/credentials 
+sudo chown coder:coder /home/coder/.aws/credentials
+```
+
+Navigate to the `./aws-linux` folder where you created your template and run the following command to put the template on your Coder instance. 
+
+```sh
+coder templates create
+```
+
+Congrats! You can now navigate to your Coder dashboard and use this Linux on AWS template to create a new workspace!
+
+## Troubleshooting
+
+If you’re not sure if Coder is running correctly as a system service, use the command `systemctl status coder` to show the status of the Coder process.
+
+To stop running Coder as a system service, run the following command: `systemctl stop coder`. 
+
+When you run ` journalctl -u coder.service -b`, there’s a chance that it may not show any logs, preventing you from seeing the url for the running Coder instance. Use the command `systemctl stop coder` to stop the Coder system service, and run the command to start Coder as a system service again. You can also use the command `systemctl restart coder` to quickly restart the service. 
```

Comment:
Where did you end up with the journalctl issue? If you really want to tell them to restart, then why not just instruct `systemctl restart coder` and remove the sentence about `systemctl stop coder`?